src/crypto/crypto.js

```diff
@@ -139,8 +139,9 @@ export default {
         const V = data_params[0].toUint8Array();
         const C = data_params[1].data;
         const d = key_params[3].toUint8Array();
+        const Q = key_params[1].toUint8Array();
```

Comment:
Sorry for being nitpicky but could you swap these two around, and in the function parameters as well? `rsa.decrypt` has the params in the same order as `key_params`, and I think it's nice for consistency.

Reply:
resolved